x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/InitializePolicyContextStep.java

```diff
@@ -72,4 +80,22 @@ public ClusterState performAction(Index index, ClusterState clusterState) {
     public boolean isRetryable() {
         return true;
     }
+
+    String policy() {
+        return policy;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
If we get the policy id from the index metadata we don't need to add this stuff either, which will be nice.